blocks/filter-group/filter-group.css

```diff
@@ -0,0 +1,81 @@
+/* !important added to override system styles to maintain cutoff 
+effect on filter buttons to indicate horizontal scroll */
+.filter-group-wrapper {
+  padding-right: 0 !important;
+
+  @media (min-width: 76rem) {
+    padding: 0 2rem !important;
+  }
+}
+
+.filter-group {
+  margin-block: 1.5rem;
+}
+
+.filter-group__list {
+  display: flex;
+  gap: 0.5rem;
+  overflow-x: auto;
+  padding-block-end: 1rem;
+  padding-inline-start: 0;
+  scrollbar-width: none; /* Firefox */
+  border-bottom: 1px solid var(--spectrum-gray-700);
+  transition: padding-inline-start 100ms ease-in-out;
+}
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+.filter-group__list::-webkit-scrollbar {
+  display: none;
+}
+
+.filter-group__list:has(.filter-group__button:first-child:focus) {
+  padding-inline-start: 0.5rem;
+}
+
+.filter-group__label {
+  margin-block-end: 0.5rem;
+  color: var(--spectrum-gray-700);
+  font-size: 1rem;
+}
+
+.filter-group__button {
+  --background-color: light-dark(
+    rgb(from var(--spectrum-black) r g b / 0.15),
+    rgb(from var(--spectrum-gray-800) r g b / 0.15)
+  );
+  display: inline-flex;
+  align-items: center;
+  font-size: 1rem;
+  justify-content: center;
+  padding: 0.625rem 1rem;
+  border-radius: var(--spectrum-corner-radius-600);
+  background-color: var(--background-color);
+  color: var(--spectrum-gray-800);
+  transition: all 100ms ease-in-out;
+  flex-shrink: 0;
+
+  & p {
+    margin: 0;
+  }
+}
+
+.filter-group__button:hover {
+  background-color: var(--background-color);
+  color: var(--spectrum-gray-800);
+  cursor: pointer;
+}
+//TODO: unify focus styles
+.filter-group__button:focus,
+.filter-group__button:focus-visible {
+  background-color: var(--background-color);
+  outline: 2px solid var(--spectrum-focus-indicator-color);
+  outline-offset: 2px;
+  color: inherit;
```

Comment:
Not sure these are working as expected 

![2025-04-22 21 30 02](https://github.com/user-attachments/assets/5035601e-855f-414f-ae5e-e60249213890)

Reply:
Good catch! Looks like this broke when I added in the  `// TODO: Unify focus styles` comment you suggested here. Forgot I wasn't in a Sass file and accidentally used // for the comment instead of the  /* ... */ CSS syntax.